app/__init__.py

```diff
@@ -20,10 +23,16 @@ def create_app():
     db.init_app(app)
     ckeditor.init_app(app)
     Bootstrap(app)
+    csrf.init_app(app)
 
     with app.app_context():
-        from . import user_views, public_views  # Import routes
-        db.create_all()  # Create sql tables for our data models
-
+        from . import user_views, public_views, error_handlers, auth, models  # Import routes
```

Comment:
Yes this is to avoid a circular import. If it is placed at the top with the other imports you will get this error ImportError: cannot import name 'db' from partially initialized module 'app' (most likely due to a circular import)

Reply:
👍  It's not a huge issue, more just an "untidy" thing. Sometimes with some thinking and maybe moving or changing a bit of code, it's possible to eliminate the circular dependency. If you want to try it, that's cool, or if you want us to look at it together some time (e.g. during our call next week), feel free.

It's good "mental practice" to be able to think about code this way and solve these issues, even though the issue itself is not that important, if that makes sense. 😅